posts/2023/10/hpy_meetup.md

```diff
@@ -0,0 +1,136 @@
+<!--
+.. title: HPy meetup and CPython core dev sprint in Brno (Oct 7-10, 2023)
+.. slug: hpy-meetup-2023
+.. date: 2023-10-10 10:00:00 UTC
+.. author: fangerer
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+We just ended our rather informal HPy meetup in Brno (Czeck Republic) which
```

Comment:
```suggestion
We just ended our rather informal HPy meetup in Brno (Czech Republic) which
```

Reply:
oops, thanks 😄 (wrote that on the train at night; I was already a bit tired)